dspace-rest/src/main/java/org/dspace/rest/filter/ItemFilterUtil.java

```diff
@@ -0,0 +1,263 @@
+		return countBitstreamMimeStartsWith(context, BundleName.ORIGINAL, item, prefix);
+	}
+	static int countBitstreamMimeStartsWith(Context context, BundleName bundleName, Item item, String prefix) {
+		int count = 0;
+        try {
+            for(Bundle bundle: item.getBundles()){
+            	if (!bundle.getName().equals(bundleName.name())){
+            		continue;
+            	}
+            	for(Bitstream bit: bundle.getBitstreams()) {
+                    if (bit.getFormat(context).getMIMEType().startsWith(prefix)) {
+                    	count++;
+                    }
+                }
+            }
+        } catch (SQLException e) {
+        }
+		return count;
+	}
+
+	static boolean hasUnsupportedBundle(Item item, String bundleList) {
+		if (bundleList == null) {
+			return false;
+		}
+		ArrayList<String> bundles = new ArrayList<String>();
+		for(String bundleName: bundleList.split(",")) {
+			bundles.add(bundleName);
+		}
+		for(Bundle bundle: item.getBundles()) {
+			if (!bundles.contains(bundle.getName())) {
+				return true;
+			}
+		}
+		return false;
+	}
+	static boolean hasOriginalBitstreamMime(Context context, Item item, String mimeList) {
+		return hasBitstreamMime(context, BundleName.ORIGINAL, item, mimeList);
+	}
+	static boolean hasBitstreamMime(Context context, BundleName bundleName, Item item, String mimeList) {
+		return countBitstreamMime(context, bundleName, item, mimeList) > 0;
+	}
+
+	static boolean hasMetadataMatch(Item item, String fieldList, Pattern regex) {
+		if (fieldList.equals("*")) {
+			for(MetadataValue md: itemService.getMetadata(item, org.dspace.content.Item.ANY, org.dspace.content.Item.ANY, org.dspace.content.Item.ANY, org.dspace.content.Item.ANY)){
+				if (regex.matcher(md.getValue()).matches()) {
+					return true;
+				}				
+			}
+		} else {
+			for(String field: fieldList.split(",")) {
+				for(MetadataValue md: itemService.getMetadataByMetadataString(item, field)){
+					if (regex.matcher(md.getValue()).matches()) {
+						return true;
+					}
+				}			
+			}			
+		}
+		
+		return false;
+	}
+
+	static boolean hasOnlyMetadataMatch(Item item, String fieldList, Pattern regex) {
+		boolean matches = false;
+		if (fieldList.equals("*")) {
+			for(MetadataValue md: itemService.getMetadata(item, org.dspace.content.Item.ANY, org.dspace.content.Item.ANY, org.dspace.content.Item.ANY, org.dspace.content.Item.ANY)){
+				if (regex.matcher(md.getValue()).matches()) {
+					matches = true;
+				} else {
+					return false;
+				}
+			}
+		} else {
+			for(String field: fieldList.split(",")) {
+				for(MetadataValue md: itemService.getMetadataByMetadataString(item, field)){
+					if (regex.matcher(md.getValue()).matches()) {
+						matches = true;
+					} else {
+						return false;
+					}
+				}			
+			}
+		}
+		return matches;
+	}
+
+	static boolean recentlyModified(Item item, int days) {
+		String accDate = itemService.getMetadata(item, "dc.date.accessioned");
```

Comment:
Recently modified is by stored in the database by default as a column on item, not in the dc.date.accessioned metadata field.